packages/fxa-content-server/app/scripts/views/support.js

```diff
@@ -112,8 +120,13 @@ const SupportView = BaseView.extend({
   onFormChange(e) {
     e.stopPropagation();
 
+    const plan = this.planEl.val();
+    const subhubPlan = this.findPlan(plan);
```

Comment:
@bbangert The subhub plan with `product_id` and `plan_id` based on the user's selection.  It is `undefined` if the user selects "Other", however.